Src/ElasticScale.Client/ShardManagement/Errors.resx

```diff
@@ -213,6 +213,9 @@
   <data name="_SqlShardMapManagerCredentials_ConnectionStringPropertyRequired" xml:space="preserve">
     <value>The required property '{0}' must be set in the connection string.</value>
   </data>
+  <data name="_SqlShardMapManagerCredentials_ConnectionStringPropertyNotAllowed" xml:space="preserve">
+    <value>The required property '{0}' must not be set in the connection string.</value>
```

Comment:
Is 'required' correct here?

Reply:
No it isn't, cut and paste issue from line above!